testing/environments/mtls/run.sh

```diff
@@ -42,6 +42,9 @@ docker pull 368821881613.dkr.ecr.us-west-2.amazonaws.com/confluentinc/cp-zookeep
 # Download latest cp-server image.
 docker pull 368821881613.dkr.ecr.us-west-2.amazonaws.com/confluentinc/cp-server:master-latest
 
+# Make sure kafka-rest is packaged.
+mvn -f "$base_dir"/kafka-rest/pom.xml -Dmaven.test.skip=true clean package
```

Comment:
nit: Do we want to skip compiling tests?
`-DskipTests` will skip running the tests while still compiling them, and will do that only negligibly slower than `-Dmaven.test.skip=true`.

Reply:
Done.